src/auth.js

```diff
@@ -4,12 +4,14 @@ var config = require('./config');
 var models = require('./models');
 var middleware = require('./middleware');
 
-function sendToken (user_id, res) {
+function sendToken (user_id, res, user) {
```

Comment:
You dont need `user_id` if you are getting `user`.  Change this and all calls to
```js
function sendToken (user, res)
```

And get by user id by `user.id` then inside `sendToken`